web/src/beta/features/Editor/Map/LayerStylePanel/Editor/StyleInterface/StyleNode/ConditionsTab.tsx

```
@@ -0,0 +1,278 @@
+      newConditions.splice(targetIndex, 0, draggedItem);
+
+      onUpdate(newConditions);
+    },
+    [conditions, onUpdate]
+  );
+
+  const handleMoveEnd = useCallback(
+    (itemIdx?: string, newIndex?: number) => {
+      if (itemIdx !== undefined && newIndex !== undefined) {
+        const parsedIndex = parseInt(itemIdx, 10);
+        if (!isNaN(parsedIndex)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)